Website/Website/App_Code/DAO/VisitorDAO.cs

```diff
@@ -111,22 +108,37 @@ public Visitor LoginVisitor(string email)
             return null;
         }
 
-        public Visitor AddVisitor(string email)
+        public void CheckIfVisitorExists(string email)
```

Comment:
Bij deze methode verwacht ik eerder een bool die aangeef of die visitor ook echt bestaat. Niet dat een visitor wordt aangemaakt.

Reply:
dit is een methode gemaakt door leon.

Reply:
deze is van mij hehe als je goed de code leest zie je dat als een gebruiker niet bestaat dat ie dan een nieuwe aanmaakt en deze terugstuurt, dus in principe gaat deze methode nooit fout (alleen als de database stuk is, maar dan logt ie gwn niet in)

Reply:
oke robert zei wat net tegen me en nu snapte ik de probleem die hij had hehe gefixt